examples/oneapi/cpp/source/kmeans/kmeans_dense_batch.cpp

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
```

Comment:
How correctly does the example solve the problem of finding clusters. Did you check? It looks very suspicious.